repository/t.json

```diff
@@ -2233,6 +2233,20 @@
 				}
 			]
 		},
+		{
+			"name": "Thread Dump Analyser",
+			"author": "Vasu",
+			"details": "https://github.com/vasugr/Thread-Dump-Analyser",
+			"readme": "https://raw.githubusercontent.com/vasugr/Thread-Dump-Analyser/main/README.md",
+			"issues": "https://github.com/vasugr/Thread-Dump-Analyser/issues",
```

Comment:
Your README and issues are in the default location, so you don't need to link to them here (ie. please remove these lines)

Reply:
Thanks. I have made the changes.